### DISCUSSION TITLE
Today widget

### DISCUSSION THREAD

**Discussion by Mahmoudsmonem** [iSoron/uhabits#1701] [Feature Requests]
First of all, thanks a lot for the good work, I really appreciate it.

An interactive widget with today view of all habits that filter/hide completed and only show remaining todos would be perfect.

Thanks

**Comment by schielb** [comment]
Yes, this would be amazing! Any day that I am set to receive a reminder for a habit (or if that habit's frequency is set to every day), it can show up on this widget on the homescreen, and so if a person finishes their task early in a day, they can just report that form the home screen.

This widget would also allow the app to serve as a daily to-do list for the regular things.
